### ISSUE TITLE
ERR_PACKAGE_PATH_NOT_EXPORTED: Package subpath './cache'

### ISSUE THREAD

**Issue by beytarovski** [hwchase17/langchainjs#661]
Hello everyone,

I'm facing an error while trying to import RedisCache for RedisClient using the following code:

```
import { RedisCache } from 'langchain/cache'
```

The error message is:

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './cache' is not defined by "exports"
```

Here are the dependencies listed in my package.json:

```
...
"dependencies": {
    "d3-dsv": "^3.0.1",
    "dotenv": "^16.0.3",
    "hnswlib-node": "^1.4.2",
    "langchain": "^0.0.49",
    "openai": "^3.2.1",
    "redis": "^4.6.5"
  },
...
```

Any help on resolving this issue would be greatly appreciated. Thank you!

**Comment by nfcampos** [comment]
Ah sorry, that is fixed in #664

**Comment by nfcampos** [comment]
The fix was released in 0.0.50
